etcdctl/ctlv3/command/del_command.go

```diff
@@ -70,6 +74,10 @@ func getDelOp(args []string) (string, []clientv3.OpOption) {
 			cobrautl.ExitWithError(cobrautl.ExitBadArgs, fmt.Errorf("too many arguments, only accept one argument when `--prefix` or `--from-key` is set"))
 		}
 		opts = append(opts, clientv3.WithRange(args[1]))
+		if !delRange {
+			fmt.Fprintln(os.Stderr, "Warning: This operations is implicitly deleting range of keys. Please add flag `--range` to make it explicit. Will sleep after v3.5.")
```

Comment:
In 3.6: It's just `Please add flag `--range` to make it explicit and avoid 2s delay`.

Reply:
Thanks. That's make sense. Fixed.

Reply:
I mean: 
 - there should be sleep 
 - the warning should say: `Please add flag --range to make it explicit and avoid 2s delay.`

Reply:
Sorry for your time. 
I fixed.